src/Sentry/SentryProcessor.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License (AFL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
+ */
+
+namespace PrestaShop\Module\PrestashopCheckout\Sentry;
+
+use Monolog\Processor\ProcessorInterface;
+
+/**
+ * Class SentryProcessor
+ */
+class SentryProcessor implements ProcessorInterface
+{
+    /**
+     * @param array $record
+     *
+     * @return array
+     */
+    public function __invoke(array $record)
+    {
+        if (!isset($record['context'])) {
+            $record['context'] = [];
+        }
+
+        $user = \Context::getContext()->employee;
```

Comment:
If you are in front office, employee is null
You should check before use object properties to avoid error

Yo be sure :
- In FO we get email from customer if exist
- In BO we get email from employee
?

Reply:
Yes, Alexis told me the same, i will change it

Reply:
check in next commit